### ISSUE TITLE
Panic when creating an egress

### ISSUE THREAD

**Issue by jlpedrosa** [livekit/livekit-server#484]
**Describe the bug**
Panic in live-kitserver side when creating a WebComposition (sample script in the bottom):

Panic on server logs:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0xc0898a]

goroutine 52 [running]:
github.com/livekit/livekit-server/pkg/telemetry.(*telemetryServiceInternal).EgressStarted(0xc000123ce0, {0x108a458, 0xc001dcb290}, 0x0)
	/workspace/pkg/telemetry/telemetryserviceinternalevents.go:260 +0xca
github.com/livekit/livekit-server/pkg/telemetry.(*telemetryService).EgressStarted.func1()
	/workspace/pkg/telemetry/telemetryservice.go:158 +0x31
github.com/livekit/livekit-server/pkg/telemetry.(*telemetryService).run(0xc000116fd8)
	/workspace/pkg/telemetry/telemetryservice.go:65 +0xa4
created by github.com/livekit/livekit-server/pkg/telemetry.NewTelemetryService
	/workspace/pkg/telemetry/telemetryservice.go:51 +0x1a5
```

**Server**
- Version: [0.15.5]
- Environment: [Minikube]
- any other information about your deployment setup

**Client**
- SDK: [golang]
- Version: [v0.9.1]

**To Reproduce**
sample code:
```
        edgeService := lksdk.NewEgressClient(config.URL, config.APIKey, config.APISecret)
	resp, err := roomService.CreateRoom(ctx, &livekit.CreateRoomRequest{
		Name:            "GolangRoom",
		EmptyTimeout:    100,
		MaxParticipants: 20,
		Metadata:        "{ }",
	})
	if err != nil {
		log.Panicf("error creating room +%v", err)
	}
	log.Printf("room created +%v", resp)

	edgeEndpoint, err := edgeService.StartWebCompositeEgress(ctx, &livekit.WebCompositeEgressRequest{
		RoomName:  roomName,
		AudioOnly: false,
		VideoOnly: false,
	})
```
**Expected behavior**
Not a panic :)  As I could see the here(https://github.com/livekit/livekit-server/blob/933b41e42a0a4cd07170a3db97db273edbc06d2e/pkg/service/egress.go#L84) info was null

**Screenshots**

**Additional context**

**Comment by frostbyte73** [comment]
Egress isn't available yet, for now use the RecordingService until the livekit-egress repo is made public. I'll put a fix in for the panic though

**Comment by davidzhao** [comment]
@jlpedrosa oops, I think you just discovered an "unreleased" feature :) This is the tricky thing about coordinating new features across repos.

We are still working on this and it should be released in the next major or minor version. In the mean time, you can use [livekit-recorder](https://github.com/livekit/livekit-recorder) and the `Record*` APIs

**Comment by jlpedrosa** [comment]
Hi Guys! a) Thannks for the quick response b) Sorry for the inconvenience.  No wonder  why I was not able to find docs :). Awesome job guys. 
I can't remeber If I saw somewhere that Recording was deprecated, that is why I went this route...

**Comment by davidzhao** [comment]
fixed in 67848a1dcb180ed9061692117c6de537d7f46a31
